packages/twenty-front/src/pages/settings/SettingsWorkspaceMembers.tsx

```diff
@@ -45,6 +60,45 @@ export const SettingsWorkspaceMembers = () => {
     setIsConfirmationModalOpen(false);
   };
 
+  const workspaceInvitations = useRecoilValue(workspaceInvitationsState);
```

Comment:
same thing here about listening to the states + let's extract handleRemoveWorkspaceInvitation to a hook like useInviteUser